src/mnemonics/electrum-words.h

```diff
@@ -103,6 +112,16 @@ namespace crypto
     bool bytes_to_words(const crypto::secret_key& src, std::string& words,
       const std::string &language_name);
 
+    /*!
+     * \brief Converts bytes (secret key) to seed words.
+     * \param  src           16-byte legacy secret key
+     * \param  words         Space delimited concatenated words get written here.
+     * \param  language_name Seed language name
+     * \return               true if successful false if not. Unsuccessful if wrong key size.
+     */
+    bool bytes_to_words(const crypto::legacy16B_secret_key& src, std::string& words, 
```

Comment:
I don't think we need this function, unless we want to promote the use of the 13-word seed which is arguably less secure than the standard 25-word seed.